tests/data/data.json

```diff
@@ -415,6 +415,107 @@
       }
     ]
   },
+  "lib5": {
+    "$schema": "https://ils.rero.ch/schema/libraries/library-v0.0.1.json",
+    "address": "Centre culturel",
+    "code": "AOSTE",
+    "email": "reroilstest+aoste@gmail.com",
+    "name": "Bibliothèque d'Aoste",
```

Comment:
The `org2` is the `district of Sion`, better to change the name of the library from `Aoste` to one of these:
`Arbaz`, `Grmisuat`, `Savièse`, ou `Vesyonnaz`

Reply:
done